test/grisu.jl

```diff
@@ -1264,122 +1305,122 @@ if status
   fill!(buffer,0);
 end
 
-status,len,point,buffer = Grisu.fastfixedtoa(Float16(1.0), 0,1, buffer)
+status,len,point = Grisu.fastfixedtoa(Float16(1.0), 0,1, buffer)
 @test "1" == bytestring(pointer(buffer))
 @test 1 == point
 fill!(buffer,0);
 
-status,len,point,buffer = Grisu.fastfixedtoa(Float16(1.0), 0,15, buffer)
+status,len,point = Grisu.fastfixedtoa(Float16(1.0), 0,15, buffer)
 @test "1" == bytestring(pointer(buffer))
 @test 1 == point
 fill!(buffer,0);
 
-status,len,point,buffer = Grisu.fastfixedtoa(Float16(1.0), 0,0, buffer)
+status,len,point = Grisu.fastfixedtoa(Float16(1.0), 0,0, buffer)
```

Comment:
Yeah, probably. I actually have the full grisu suite setup that way (100K tests for each major grisu path). These are the smaller batches that test each code branch individually, so it's typically nicer to run these separately and see exact test/line number that failed and then run the full batch. We could probably run the full test suite now that things have sped up so much. It would probably be 3-5 seconds total to run the tests, though I've found that actually loading the test instances takes the bulk of the time (~10 seconds or so).